package/cfg/cis-1.4/etcd.yaml

```diff
@@ -0,0 +1,229 @@
+          chmod 700 /var/lib/etcd
+        scored: true
+
+      - id: 1.4.12
+        text: "Ensure that the etcd data directory ownership is set to etcd:etcd (Scored)"
+        audit: ps -ef | grep $etcdbin | grep -- --data-dir | sed 's%.*data-dir[= ]\([^ ]*\).*%\1%' | xargs stat -c %U:%G
+        tests:
+          test_items:
+          - flag: "etcd:etcd"
+            set: true
+        remediation: |
+          On the etcd server node, get the etcd data directory, passed as an argument --data-dir ,
+          from the below command:
+          ps -ef | grep $etcdbin
+          Run the below command (based on the etcd data directory found above). For example,
+          chown etcd:etcd /var/lib/etcd
+        scored: true
+
+      - id: 1.4.19
+        text: "Ensure that the Kubernetes PKI directory and file ownership is set to root:root (Scored)"
+        audit: "ls -laR /etc/kubernetes/pki/"
+        type: "manual"
+        tests:
+          test_items:
+          - flag: "root root"
+            compare:
+              op: eq
+              value: "root root"
+            set: true
+        remediation: |
+          [Manual test]
+          Run the below command (based on the file location on your system) on the master node.
+          For example, chown -R root:root /etc/kubernetes/pki/
+        scored: true
+
+      - id: 1.4.20
+        text: "Ensure that the Kubernetes PKI certificate file permissions are set to 644 or more restrictive (Scored)"
+        audit: "stat -c %n\ %a /etc/kubernetes/pki/*.crt"
+        type: "manual"
+        tests:
+          bin_op: or
+          test_items:
+            - flag: "644"
+              compare:
+                op: eq
+                value: "644"
+              set: true
+            - flag: "640"
+              compare:
+                op: eq
+                value: "640"
+              set: true
+            - flag: "600"
+              compare:
+                op: eq
+                value: "600"
+              set: true
+        remediation: |
+          [Manual test]
+          Run the below command (based on the file location on your system) on the master node.
+          For example, chmod -R 644 /etc/kubernetes/pki/*.crt
+        scored: true
+
+      - id: 1.4.21
+        text: "Ensure that the Kubernetes PKI key file permissions are set to 600 or more restrictive (Scored)"
+        audit: "stat -c %n\ %a /etc/kubernetes/pki/*.key"
+        type: "manual"
+        tests:
+          test_items:
+            - flag: "600"
+              compare:
+                op: eq
+                value: "600"
+              set: true
+        remediation: |
+          [Manual test]
+          Run the below command (based on the file location on your system) on the master node.
+          For example, chmod -R 600 /etc/kubernetes/pki/*.key
+        scored: true
+
+      - id: 1.5
+        text: "etcd"
+        checks:
```

Comment:
indent everything left, currently falls in `groups[0].checks` instead of `groups[1]`

Reply:
good catch, thanks